client/src/components/new-request/submit-request/entity-cannot-be-auto-analyzed.vue

```diff
@@ -13,7 +13,7 @@
         <v-col v-for="(box, i) in boxes" :key="'box-'+i">
           <v-container class="copy-small text-left" :class="box.class">
             <v-row align-content="space-between" style="height: 100%">
-              <v-col class="h5 py-0"><v-icon class="pr-2 colour-p-blue-text">mdi-information</v-icon>
+              <v-col class="h5 py-0"><v-icon class="pr-2 colour-p-blue-text">info</v-icon>
```

Comment:
Please change this back to a MDI icon. We will transition everything to MDI icons and then stop using MD icons (for consistency between apps).

Reply:
ooops... that was me careless restoring from local history.  I meant to remove the icons altogether